tests/agent_features/test_log_events.py

```diff
@@ -102,10 +118,10 @@ def exercise_record_log_event():
         "message": "serialized_attributes",
         "context.str_attr": "Value",
         "context.bytes_attr": b"value",
-        "context.int_attr": "1",
-        "context.dict_attr": '{"key":"value"}',
+        "context.int_attr": 1,
+        "context.dict_attr": "{'key': 'value'}",
         "context.non_serializable_attr": "<NonSerializableObject object>",
-        "context.non_printable_attr": "<unprintable NonPrintableObject object>",
+        "context.attr_value_too_long": "*" * 255,
```

Comment:
Looks like this line replaced the previous instead of adding a new line.

Reply:
Yeah, the `context.non_printable_attr` moved to a lower section for forgone attrs. These shouldn't be included anymore to match the implementation for existing attribute systems where a thrown exception drops the attribute (or all attributes).